### DISCUSSION TITLE
Orion Compatibility

### DISCUSSION THREAD

**Discussion by NadaKamel03** [leapmotion/UnityModules#1306] [Q&A] [answered]
Is this version of the plugin compatible with the Orion Hand Tracking Software version 4.1?

**Comment by MattGrayUL** [comment, marked as answer]
Hi @NadaKamel03

Unfortunately support for Orion tracking stopped at version 5.0 of the Unity Plugin

This means the last Plugin Version that supports Orion is 4.X, the latest fo which is [available here](https://github.com/ultraleap/UnityPlugin/releases/tag/UM-4.8.0)

Was there any particular reason you are unable to use Gemini (v5.X) tracking?

**Reply by NadaKamel03** [reply]
I have attempted to install Gemini (v5.x) tracking but my device cannot be detected for some reason. I thought the issue was the device but it connected perfectly well to Orion.

**Reply by MattGrayUL** [reply]
Ah that's not good, have you contacted [support](https://support.leapmotion.com/hc/en-us/requests/new) to let them know?

If you wish to continue with Orion, does the above link (version 4.8 of the Unity Plugin) work for you? :)
